docs/deployment/linux/ubuntu.txt

```diff
@@ -7,25 +7,23 @@ Run CrateDB on Ubuntu
 CrateDB maintains packages for the following Ubuntu versions:
 
 - `Bionic Beaver`_ (18.04)
-- `Artful Aardvark`_ (17.10)
-- `Zesty Zapus`_ (17.04)
-- `Yakkety Yak`_ (16.10)
 - `Xenial Xerus`_ (16.04)
 - `Trusty Tahr`_ (14.04)
 
 .. _Bionic Beaver: https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes
-.. _Artful Aardvark: https://wiki.ubuntu.com/ArtfulAardvark/ReleaseNotes
-.. _Zesty Zapus: https://wiki.ubuntu.com/ZestyZapus/ReleaseNotes
-.. _Yakkety Yak: https://wiki.ubuntu.com/YakketyYak/ReleaseNotes
 .. _Xenial Xerus: https://wiki.ubuntu.com/XenialXerus/ReleaseNotes
 .. _Trusty Tahr: https://wiki.ubuntu.com/TrustyTahr/ReleaseNotes
 
 .. NOTE::
 
    CrateDB requires Java 8 or higher.
 
-   To run CrateDB on Ubuntu releases older than Trusty Tahr 14.10, you will need
-   to :ref:`install Java 8 from a 3rd party repository <debian-wheezy>`.
+   To run CrateDB on Ubuntu releases older than 16.04, you will need to install
+   Java from a 3rd party repository. This can be done by adding the openjdk ppa:
```

Comment:
"third-party"

"openjdk PPA package"

Reply:
Not sure if "package" is correct. It's a repository. PPA stand for Personal Package Archive, so `PPA package` sounds a bit like `HTTP Protocol`.

Reply:
oh you're right yes! just "PPA"! thx